Source/Urho3D/Core/ProcessUtils.cpp

```diff
@@ -540,7 +540,7 @@ String GetLoginName()
 {
 #if defined(__linux__) && !defined(__ANDROID__)
     struct passwd *p = getpwuid(getuid());
-    if (p) 
+    if (p != NULL) 

```

Comment:
While this is indeed violated the coding convention, I saw that comparing to NULL (`!= NULL`) would make it easier for the later plan of porting the code to C++11 (I saw a thread discussing about the move to C++11 somewhere if I remember right), you can just do a simple find & replace (`!= NULL` with `!= nullptr`). With only `if (p)` instead of `if (p != NULL)`, it's a little harder.